ducktape/command_line/main.py

```diff
@@ -184,5 +186,6 @@ def main():
     reporter = HTMLSummaryReporter(test_results)
     reporter.report()
 
+    update_latest_symlink(args.results_root, results_dir)
```

Comment:
Is there any chance this could be lost to an exception? I obviously prefer the new behavior, but want to make sure it works well in case of failures too. Since it's deferred until the very end now, unexpected errors could cause the update to not occur.

Are we confident enough of the rest of the code in here that is not in a `try`/`catch` block that we don't need to put this in a `finally` clause? Or do we just think that if something does throw an exception, the issue is obvious and we should ignore the `latest` directory anyway?